pitop/common/pt_os.py

```diff
@@ -1,19 +1,74 @@
+from dataclasses import dataclass, asdict
 from os import path
 from getpass import getuser
 
 
+ptissue_path = "/boot/pt-issue.txt"
+legacy_ptissue_path = "/etc/pt-issue"
+
+
+@dataclass
+class PitopOsBuildInfoV1:
+    schema_version: str
+    build_type: str
+    date: str
+    build_time_os_version: str
+    build_run_number: str
+    commit: str
+
+    @classmethod
+    def from_os_build_info(cls, build_info):
+        DELIMITER: str = "_"
+        LOOKUP = {
+            "S": "schema_version",
+            "B": "build_type",
+            "D": "date",
+            "P": "build_time_os_version",
+            "R": "build_run_number",
+            "#": "commit",
```

Comment:
I think these could be more standardised, they are all based on build time, so:
date -> build_date
build_time_os_version -> build_os_version
commit -> build_commit

I know in this context it might be implied that date and commit relate to a build, and it's less clear for os_version, but thinking from the perspective of potentially passing these as parameters to the update check api it's not clear why they differ.